### PR TITLE
Vs-eng-update-lockdown-attack-lv-381

### PR DESCRIPTION
**Pull request by RyanASwanson**
https://bradleycapstone.atlassian.net/jira/software/projects/LV/boards/32/backlog?assignee=6318bb796856bdd60aa045de&selectedIssue=LV-381
A lot of stuff being renamed. Some minor adjustments to the lockdown attack 
Lifetime functionality removed

### CODE REVIEW

**Review by AlecatEnjoyer — APPROVED** [review]
approve

**Review by astapay — APPROVED** [review]
yippee!

File: Assets/GameAssets/Boss/BossScripts/Attacks/Lockdown/LockdownAttack.cs
Comment (ASLH68) [inline Assets/GameAssets/Boss/BossScripts/Attacks/Lockdown/LockdownAttack.cs:1]: Add self as contributor

**Review by ASLH68 — CHANGES_REQUESTED** [review]

File: Assets/GameAssets/Boss/BossScripts/Attacks/Lockdown/LockdownAttackController.cs [hunk 1]
```diff
@@ -0,0 +1,151 @@
+/*****************************************************************************
+// File Name :         RoomTongueAttack.cs
```
Comment (ASLH68) [inline Assets/GameAssets/Boss/BossScripts/Attacks/Lockdown/LockdownAttackController.cs:2]: Update file name

File: Assets/GameAssets/Boss/BossScripts/Attacks/Lockdown/LockdownAttackController.cs [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    private LockdownAttackEnemyController[] _lockdownEnemyControllers;
+
+    /// <summary>
+    /// Called when the attack ends to destroy all spawned enemies
+    /// </summary>
+    public static UnityEvent DestroyAllEnemies {get; private set;} = new();
+
+    #region Enable & Action/Event Subscriptions
+
+    //NOTE: Not sure why this isn't done in a base class but ok. Not changing it now to avoid issues with other attacks
```
Comment (ASLH68) [inline Assets/GameAssets/Boss/BossScripts/Attacks/Lockdown/LockdownAttackController.cs:40]: Can remove this comment imo

File: Assets/GameAssets/Boss/BossScripts/Attacks/Lockdown/LockdownAttackController.cs [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+    /// <summary>
+    /// Unsubscribes to events on disable
+    /// </summary>
+    private void OnDisable()
+    {
+        UnsubscribeToEvents();
+    }
+
+    //NOTE: Not sure why this is an override and isn't done in the base class.
```
Comment (ASLH68) [inline Assets/GameAssets/Boss/BossScripts/Attacks/Lockdown/LockdownAttackController.cs:58]: same here w the comment, doesnt add understanding to the file as is

File: Assets/GameAssets/Boss/BossScripts/Attacks/Lockdown/LockdownAttackEnemyController.cs [hunk 4]
```diff
@@ -0,0 +1,156 @@
+/*****************************************************************************
+// File Name :         PatrolEnemySpawner.cs
+// Author :            Tommy Roberts
+// Contributors:       Andrea Swihart-DeCoster
```
Comment (ASLH68) [inline Assets/GameAssets/Boss/BossScripts/Attacks/Lockdown/LockdownAttackEnemyController.cs:4]: add self as contributor

**Review by ASLH68 — APPROVED** [review]

**Pull request merged**